packages/v8-deopt-parser/src/optimizationStateParsers.js

```diff
@@ -43,6 +47,8 @@ export function severityOfOptimizationState(state) {
 			return MIN_SEVERITY + 1;
 		case Profile.CodeState.OPTIMIZED:
 			return MIN_SEVERITY;
+		case Profile.CodeState.BASELINE:
+			return MIN_SEVERITY + 2; // Not entirely sure about this
```

Comment:
I forget, but I think UNKNOWN_SEVERITY will have this show up as a grey in the UI? Perhaps until we know for certain if it is good or bad we can return that?

Reply:
Good idea! That's makes more sense than trying to tie it to another level 👍